pyproject.toml

```diff
@@ -9,28 +9,29 @@ homepage = "https://signbank.nzsl.nz/"
 repository = "https://github.com/ODNZSL/NZSL-signbank"
 
 [tool.poetry.dependencies]
-python = "^3.9"
-django = "~3.2.25"
-django-bootstrap3 = "*"
-django-contrib-comments = "*"
-django-debug-toolbar = "*"
-django-guardian = "*"
-django-modeltranslation = "*"
-django-notifications-hq = "*"
-django-queryset-csv = "*"
-django-registration = "*"
-django-reversion = "*"
-django-storages = "*"
-django-summernote = "*"
-django-tagging = "*"
-boto3 = "*"
-certifi = "*"
-dj-database-url = "*"
-gunicorn = "*"
-psycopg2 = "*"
-sentry-sdk = "*"
-whitenoise = "*"
+python="^3.9"
+django="3.2.25"
+django-tagging="0.5.0"
+django-reversion="4.0.0"
+django-bootstrap3="21.2"
+django-summernote="0.8.11.6"
+django-modeltranslation="0.18.2"
+django-registration="3.1.2"
+django-contrib-comments="1.9.2"
+django-queryset-csv="1.1.0"
+django-debug-toolbar="3.2.2"
+django-guardian="2.2.0"
+django-notifications-hq="1.6.0"
+dj-database-url="0.5.0"
+boto3="1.22.8"
+django-storages="1.12.1"
+psycopg2="2.8.6"
+certifi="2023.7.22"
+sentry-sdk="1.14.0"
+gunicorn = "22.0.0"
+whitenoise="6.2.0"
```

Comment:
issue: can you restore a space around the `=`s?

Reply:
Sure!

Reply:
Done.